lib/cs_guide_web/controllers/static_page_controller.ex

```diff
@@ -0,0 +1,81 @@
+defmodule CsGuideWeb.StaticPageController do
+  use CsGuideWeb, :controller
+
+  alias CsGuide.StaticPage
+
+  import Ecto.Query, only: [from: 2, subquery: 1]
+
+  def index(conn, _params) do
+    static_pages = StaticPage.all()
+
+    render(conn, "index.html", static_pages: static_pages)
+  end
+
+  def new(conn, _params) do
+    changeset = StaticPage.changeset(%StaticPage{}, %{})
+    render(conn, "new.html", changeset: changeset)
+  end
+
+  def create(conn, %{"static_page" => static_page_params}) do
+    case %StaticPage{} |> StaticPage.changeset(static_page_params) |> StaticPage.insert() do
+      {:ok, static_page} ->
+        conn
+        |> put_flash(:info, "Static Page created successfully.")
+        |> redirect(to: static_page_path(conn, :index))
+
+      {:error, %Ecto.Changeset{} = changeset} ->
+        render(conn, "new.html", changeset: changeset)
+    end
+  end
+
+  def show(conn, %{"id" => id}) do
```

Comment:
`id` here should be `page_name`, so it matches what we've got in the router 
```
get("/:page_name", StaticPageController, :show)
```
https://github.com/club-soda/club-soda-guide/pull/243/files#diff-aab95ee0d4d80e06aafb52570a736698R38